templates/corporate/hello.html

```diff
@@ -1,562 +1,402 @@
-{% extends "zerver/portico.html" %}
-{% set entrypoint = "landing-page" %}
-
-{% set PAGE_TITLE = "Zulip: Open-source team chat with topic-based threading" %}
-
-{% set PAGE_DESCRIPTION = "Zulip is the only modern team chat app that is
-  designed for both live and asynchronous conversations. Organized to help you
-  collaborate productively and efficiently." %}
-
+{% extends "zerver/base.html" %}
+{% set entrypoint = "landing-page-hello" %}
 {% block customhead %}
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-<style>
-    .portico-page {
-    padding-bottom: 0px;
-    }
-</style>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
+    <title>Zulip — organized team chat</title>
+    <!-- Google / search engine tags -->
+    <meta name="description" content="Zulip is the only modern team chat app that is designed for both live and asynchronous conversations. Organized to help you collaborate productively and efficiently." />
+    <!-- Open Graph / Facebook / Twitter meta tags -->
+    <meta property="og:url" content="https://zulipchat.com/" />
+    <meta property="og:type" content="website" />
+    <meta property="og:site_name" content="Zulip" />
+    <meta property="og:title" content="Zulip: Open-source team chat with topic-based threading" />
+    <meta property="og:description" content="Zulip is the only modern team chat app that is designed for both live and asynchronous conversations. Organized to help you collaborate productively and efficiently." />
+    <meta property="og:image" content="https://zulipchat.com/static/images/logo/zulip-icon-128x128.png" />
+    <meta name="twitter:card" content="summary" />
+    <!--   favicon -->
+    <link rel="icon" href="{{ static('images/landing-page/hello/favicon.ico') }}" sizes="any"/> <!-- 32×32 -->
+    <link rel="apple-touch-icon" href="{{ static('images/landing-page/hello/apple-touch-icon.png') }}"/> <!-- 180×180 -->
+    <!-- font docs  https://developers.google.com/fonts/docs/css2 -->
+    <link rel="preconnect" href="https://fonts.googleapis.com"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300..700&family=Source+Sans+3:wght@300..700&display=block" as="style" rel="preload"/>
```

Comment:
We generally have not made a habit of using external sources for fonts; we should bundle the fonts via webpack as do for everything else.

Reply:
We should try to merge #24833 before this then.

Reply:
Fixed but for open-sans, we are using `font-weight: 350` in a few places which [our package](https://www.npmjs.com/package/@fontsource/open-sans) doesn't support. So, I need @terpimost to confirm if we should change them to `300` or `400`. (Likely 400)

Reply:
Done.